www/src/components/layer-model/layer-model.js

```diff
@@ -116,7 +85,17 @@ const LayerModel = ({ initialLayer = `Content` }) => {
     }
   }, [selected])
   return (
-    <>
+    <div 
+      sx={{
+        borderRadius: 3,
+        border: t => `1px solid ${t.colors.pullquote.borderColor}`,
```

Comment:
@laurieontech LGTM, at least for now, IMO this shouldn't block this PR — the design tokens (and their docs) need a lot more work, and IMO it's not bad to have pointers like this, and start thinking about adding another set of tokens at a lower level, or rename the existing `pullquote` … while shipping things.

Reply:
Sounds good! Appreciate you taking a look!

Reply:
Actually, we might get away with using `colors.ui.border`, lemme have a look real quick … ✌️

Reply:
The pullquote was used because of the light/dark mode switch. Can that be accomplished with border color?

Reply:
Yep, also has a dark mode equivalent.